package.json

```diff
@@ -1,13 +1,16 @@
 {
-  "name": "lforms",
-  "version": "33.4.2",
+  "name": "@elimuinformatics/lforms",
+  "version": "2.0.0",
   "keywords": [
     "fhir",
     "Questionnaire",
     "SDC"
   ],
-  "repository": "github:lhncbc/lforms",
-  "license": "See LICENSE.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lhncbc/lforms.git"
```

Comment:
should this URL point to our fork?

Reply:
No its not

Need to change with  `https://github.com/elimuinformatics/lforms.git`

will update shortly

Reply:
Updated  ...